_pages/contributing.md

```diff
@@ -333,6 +340,12 @@ git switch new-feature # Ensure we are on the 'new-feature' branch.
 git push --set-upstream origin new-feature
 ```
 
+If this is your first time pushing to your forked gem5 repo, you might encounter an error stating that GitHub no longer accepts account passwords when authenticating Git operations. To resolve this issue, generate a personal access token at https://github.com/settings/tokens and then execute the following command:
+
+```sh
+git remote set-url origin https://<githubtoken>@github.com/<username>/<repositoryname>.git
+```
+
```

Comment:
I suggest using this documentation instead: https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories#switching-remote-urls-from-https-to-ssh

If you think it's reasonable, I would rather just point to github's documentation instead of replicating it here.

Reply:
@powerjg, I pushed the change. Please let me know if there is anything else I should edit.

Reply:
lgtm. Thanks!